jobs/buildhub/to_kinto.py

```diff
@@ -46,6 +46,11 @@ def fetch_existing(client, cache_file=PREVIOUS_DUMP_FILENAME):
     """Fetch all records since last run. A JSON file on disk is used to store
     records from previous run.
     """
+    cache_file = cache_file.format(
+        server="".join(c for c in client.session.server_url if c.isalnum()),
```

Comment:
How will that lookslike? `httpbuildhubstagemozawsnetv1` ?

Reply:
yep

Reply:
Maybe the hostname would be more readable?

Reply:
why would we need it to be more readable?